test/VoteDelegate.t.sol

```diff
@@ -141,6 +141,74 @@ contract VoteDelegateTest is DssTest {
         assertEq(proxy.stake(address(delegator2)), 0);
     }
 
+    function testReserveHatch() public {
+        vm.prank(delegate); gov.approve(address(proxy), type(uint256).max);
+
+        assertEq(gov.balanceOf(address(delegate)), 100 ether);
+        assertEq(proxy.hatchTrigger(), 0);
+
+        vm.prank(delegate); proxy.lock(10 ether);              // can lock
+        vm.prank(delegate); proxy.reserveHatch();              // can reserve hatch
```

Comment:
I would maybe not prank any account when calling `reserveHatch()` since this is a permissionless function.

Reply:
good point, done